packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -185,6 +192,7 @@ export class OdspDocumentService implements IDocumentService {
                     }
                     throw new Error("Disconnected while uploading summary (attempt to perform flush())");
                 },
+                () => this.relayServiceSessionId,
```

Comment:
See my other comment - we should assert here that it's never undefined and change types downstream to accept string only.